### ISSUE TITLE
Domain ideas

### ISSUE THREAD

**Issue by MrGKanev** [IEEE-Student-Branch-of-Uni-Ruse/website#15]
For now, we are using [sb-ieee.uni-ruse.bg/](https://sb-ieee.uni-ruse.bg/), but because of "security" limitations given by the IT team of the university we are searching for ideas on domain names. We are open to ideas, but please meet as many requirements as possible.

Requirements:
* it should have "sb" and "IEEE" somewhere
* Short
* Understandable 
* Easy to remember 
* Not being domain.bg - if possible

Example: College of Engineering, Chengannur Alappuzha, Kerala - https://cecieee.org/
IEEE University of Rajshahi Student Branch - https://ieeerusb.org/

**Comment by atilapan** [comment]
From the given example, I am suggesting for the domain name: `uniruseieee` and pick a proper TLD.

**Comment by MrGKanev** [comment]
@atilapan the name may not be as in the example mentioned above. I provided it only for the concept. Most of the IEEE student branches are a folder (/IEEE-branch) to the main domain or subdomains. - or at least the ones I found.

P.S. We haven't talked about how we will correspond on official places and I did it in English for my "easier sake". So you can talk in Bulgarian if you want. 😆

**Comment by MrGKanev** [comment]
My ideas are:
* rusieee.com - meaning: uni-ruse student IEEE - personally I like it because of its Bulgarian reading :hear_no_evil:
* sbieee-ur.com - meaning:  Student branch IEEE - uni ruse

**Comment by atilapan** [comment]
I totally agree with the first suggestion, it's really funny and catchy. (`Русиии`)
